packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -132,12 +132,9 @@ class CreateCommand extends FlutterCommand {
       defaultsTo: 'kotlin',
       allowed: <String>['java', 'kotlin'],
     );
-    // TODO(egarciad): Remove this flag. https://github.com/flutter/flutter/issues/52363
     argParser.addFlag(
-      'androidx',
```

Comment:
cc @blasten To make sure it is ready to be removed.

Reply:
it should be fine at this point